src/templates/finops-hub/modules/scripts/IngestionSetup.kql

```diff
@@ -80,6 +80,31 @@ HubScopes()
     IconUri:                  string
 )
 
+// parse_resourceid
```

Comment:
Can you move this to be after Services so all the open data tables are together? Also, remove it from the common file.